.storybook/main.js

```diff
@@ -1,8 +1,8 @@
 module.exports = {
-  stories: ["../examples/**/*.stories.(js|md)"],
+  stories: ["../examples/**/*.stories.@(js|md)"],
```

Comment:
What is the @ for?

Reply:
You can read about it here: https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#correct-globs-in-mainjs